src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1280,6 +1281,31 @@ GUI_TEST_CLASS_DEFINITION(test_7460) {
     GTUtilsTaskTreeView::waitTaskFinished(os, 10000);  // Check that there is no long-running active tasks.
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7465) {
+    //1. Open workflow sample "Align sequences with MUSCLE"
+    //Expected state: wizard has appeared.
+    class AlignSequencesWithMuscleWizardFiller : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus &os) {
+            QWidget *wizard = GTWidget::getActiveModalWidget(os);
+            GTWidget::clickWindowTitle(os, wizard);
+
+            //2. Set file with many (~1200) sequences as input file and run workflow
+            GTUtilsWizard::setInputFiles(os, {{QFileInfo(testDir + "_common_data/fastq/lymph.fastq").absoluteFilePath()}});
+            //GTUtilsWizard::setParameter(os, "Reference", QFileInfo(testDir + "_common_data/fastq/lymph.fastq").absoluteFilePath());
+            GTUtilsWizard::clickButton(os, GTUtilsWizard::Next);
+            GTUtilsWizard::clickButton(os, GTUtilsWizard::Run);
+        }
+    };
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(os, "Align Sequences with MUSCLE Wizard", new AlignSequencesWithMuscleWizardFiller));
+    GTUtilsWorkflowDesigner::openWorkflowDesigner(os);
```

Comment:
Do you really need a workflow or even a GUI test? Once we test an algorithm only why not using XML?

Reply:
I think it is possible to make this test as XML, but i dont see any advantages between this options, XML won't be faster.

Reply:
You said "Opening this file is the most time consuming operation"
Is it because of the slow FASTQ or a slow UI?

Reply:
Because of slow FASTQ reading.